### ISSUE TITLE
Does houdini already supports windows?

### ISSUE THREAD

**Issue by outerlook** [AlecAivazis/houdini#70]
Hi!

While trying to setup houdini, I got this error on `houdini generate`

```
(node:12660) UnhandledPromiseRejectionWarning: Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:782:11)
    at Loader.resolve (internal/modules/esm/loader.js:88:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:241:28)
    at Loader.import (internal/modules/esm/loader.js:176:28)
    at importModuleDynamically (internal/modules/esm/translators.js:114:35)
    at exports.importModuleDynamicallyCallback (internal/process/esm_loader.js:30:14)
    at file:///C:/.../node_modules/houdini/build/cmd.js:937:49
    at step (file:///C:/.../node_modules/houdini/build/cmd.js:346:23)
    at Object.next (file:///C:/.../node_modules/houdini/build/cmd.js:327:53)
    at file:///C:/.../node_modules/houdini/build/cmd.js:320:71
```

some of my devdeps
```
		"@sveltejs/kit": "1.0.0-next.107",
		"houdini": "^0.5.1-kit.6",
		"houdini-preprocess": "^0.5.1-kit.6",
		"typescript": "4.3.1-rc"
		"svelte": "3.38.2",
```

Does this error means it isn't focused on windows yet, or is it something wrong on my side?

**Comment by AlecAivazis** [comment]
Hey @outerlook - thanks for opening up this ticket

It looks like that error is caused when trying to load the config file. I haven't tested the sveltekit branch on the windows terminals so im not **totally** surprised there are some problems. I'll poke around tonight and see if I can spot the problem

**Comment by AlecAivazis** [comment]
I think this is now fixed on version `0.5.1-kit.7`
